docs/first-steps/management-account.md

```diff
@@ -0,0 +1,152 @@
+# Orchestrate the Management account
+
+Finally we reach the point in which you'll get to actually create the infrastructure in our AWS environment.
```

Comment:
This doesn't sound right: "you'll get to actually create the infrastructure in **our** AWS environment"

Reply:
Good catch, that's actually a typo, it should be **your** AWS environment.

Reply:
Agree to this point, it should be "in **your** AWS environment"

Reply:
Done!